src/main/java/frc/robot/Robot.java

```diff
@@ -20,6 +21,13 @@
  */
 public class Robot extends TimedRobot {
   public static final boolean isCompetition = true;
+
+  //RoboRIO serial numbers
+  public static final String competitionRobotSerial = "03161743";
```

Comment:
Design choice: This is perfectly fine as is, but consider factoring out into function as it may make top of Robot class a bit cleaner.